packages/components/src/components/Icon/icons.ts

```diff
@@ -81,6 +81,8 @@ export const ICONS = {
     CALENDAR: require('../../images/icons/calendar.svg'),
     PDF: require('../../images/icons/pdf.svg'),
     CSV: require('../../images/icons/csv.svg'),
+    BINARY: require('../../images/icons/binary.svg'),
```

Comment:
nit:
```suggestion
    BIN: require('../../images/icons/bin.svg'),
```
`PDF` and `CSV` are file extensions. Binary too, so let's name it `BIN` to make it consistent

Reply:
1. Bin also means 'trash bin', which could be kind of confusing.
2. PDF and CSV are widely used names for file types. Not sure if someone is using 'bin' for binary files.
3. Not every binary file in the future must have the .bin extension.
3. Just below there is an IMAGE icon, which also doesn't point to a specific file extension.

Reply:
Okay resolving

> Just below there is an IMAGE icon, which also doesn't point to a specific file extension.
I know. I did not mention it because IMAGE is not a file extension